.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: test
+
+on:
+  push:
+    branches: [ master, gh-actions ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # 2.7 not supported
+        # 3.5 not supported
```

Comment:
i think this needs to be documented somewhere.

failing 3.5 run here: https://github.com/williballenthin/speakeasy/runs/1032519414?check_suite_focus=true

Reply:
nvm, its specified here: https://github.com/fireeye/speakeasy/blob/master/setup.py#L36